actioncable/app/javascript/action_cable/consumer.js

```diff
@@ -32,6 +32,14 @@ export default class Consumer {
     this.url = url
     this.subscriptions = new Subscriptions(this)
     this.connection = new Connection(this)
+
+    Object.defineProperty(this, 'url', {
+      get: function() {
+        const token = Storage.get('token');
+        const email = Storage.get('email');
```

Comment:
What is `Storage`? And, presuming `Storage` exists somehow, how do the token an email values get defined?

<img width="362" alt="screen shot 2018-12-05 at 9 43 55 am" src="https://user-images.githubusercontent.com/5355/49520929-4d33ca00-f872-11e8-9366-6b1e692e6792.png">